### PR TITLE
docs: [DENA-600] CockroachDB README using shared-kustomize-bases README standard

### PR DESCRIPTION
**Pull request by MarcinGinszt**

### CODE REVIEW

File: cockroachdb/README.md [hunk 1]
```diff
@@ -1,54 +1,30 @@
...
-This repo uses tags to manage versions, these tags have two components:
+There are two manifest directories - `manifests-cfssl`, and `manifests-cert-manager`.
 
-  - The version of the `cockroachdb/cockroach` image the manifests are using
-  - An internal version to track changes to anything besides the version of
-    CockroachDB
+- `manifests-cfssl` are **not recommended** for fresh install - they contain the manifests that demand
+  certificate authority deployed within the namespace. For more information,
+  see [CFSSL README](manifests-cfssl/CFSSL_README.md)
+  - `manifests-cert-manager` are using Kubernetes cert-manager controller,
```
Comment (sbuliarca) [inline cockroachdb/README.md:17]: I think we should put `manifests-cert-manager` as the first entry, and call out that it is recommended, and call the cfssl one deprecated

File: cockroachdb/manifests-cfssl/backup-init-job.yaml
[the remark this answers is not in the document]
Comment (sbuliarca) [reply cockroachdb/manifests-cfssl/backup-init-job.yaml:1]: I don't think people know about this base in here, so I think it's highly unlikely to be used yet

File: cockroachdb/README.md [hunk 2]
```diff
@@ -1,54 +1,30 @@
-** Work in progress! State of this manifest should change in next days**
-# Cockroachdb
+# Redis
```
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply cockroachdb/README.md]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-cf57986444834382c6724834a6e2ed41af60da7becb7a6cd51066fe0ed4ea47fR1)

File: cockroachdb/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply cockroachdb/README.md]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-cf57986444834382c6724834a6e2ed41af60da7becb7a6cd51066fe0ed4ea47fR14)

File: cockroachdb/manifests-cfssl/backup-init-job.yaml
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply cockroachdb/manifests-cfssl/backup-init-job.yaml:1]: @matthewhughes-uw you are right, there were some remaining references, [fixed](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-9d9fb3fa7436209bfb820a1122fdc88b84d337d680df6273b4c5e702c3372a58R158)

File: cockroachdb/README.md [hunk 3]
```diff
@@ -1,54 +1,30 @@
...
-
-### Versioning
-
-This repo uses tags to manage versions, these tags have two components:
+There are two manifest directories - `manifests-cfssl`, and `manifests-cert-manager`.
 
-  - The version of the `cockroachdb/cockroach` image the manifests are using
-  - An internal version to track changes to anything besides the version of
-    CockroachDB
+- `manifests-cfssl` are **not recommended** for fresh install - they contain the manifests that demand
```
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply cockroachdb/README.md:17]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-cf57986444834382c6724834a6e2ed41af60da7becb7a6cd51066fe0ed4ea47fR14)

File: cockroachdb/README.md (hunk 1 above)
Comment (MarcinGinszt) [reply cockroachdb/README.md]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-cf57986444834382c6724834a6e2ed41af60da7becb7a6cd51066fe0ed4ea47fR14)

File: cockroachdb/README.md [hunk 4]
```diff
@@ -75,8 +51,47 @@ This can be achieved by:
...
-### Architecture
+## Metrics
+
+There are Grafana dashboards prepared for the Cockroachdb.
+They can be found using the name "cockroach overview",
+e.g. [for `prod-aws`](https://grafana.prod.aws.uw.systems/d/ddnrjgg8eby80e/cockroachdb-overview?orgId=1).
+
+## Backup/ migration
+
+Our CockroachDB is configured with backup by default.
```
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply cockroachdb/README.md:62]: [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/31b69e21a87427b145d8f5d824584895b9e6647a#diff-cf57986444834382c6724834a6e2ed41af60da7becb7a6cd51066fe0ed4ea47fR63)

File: cockroachdb/manifests-cfssl/backup-init-job.yaml
[the remark this answers is not in the document]
Comment (MarcinGinszt) [reply cockroachdb/manifests-cfssl/backup-init-job.yaml:1]: This broke the kustomize build, so I added [follow up commit changing tags](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/55/commits/3abcda0213f7bd48a29145556f26c06bd8d745a0)

**Review by sbuliarca — APPROVED** [review]
Nicely done!

**Pull request merged**